rules/network/discovery_potential_port_scan_detected.toml

```diff
@@ -4,16 +4,16 @@ integration = ["endpoint", "network_traffic"]
 maturity = "production"
 min_stack_comments = "New fields added: required_fields, related_integrations, setup"
 min_stack_version = "8.3.0"
-updated_date = "2023/05/26"
+updated_date = "2023/07/14"
 
 [rule]
```

Comment:
This one doesn't have `auditbeat-*` added as an index. Is this on purpose?

Reply:
Another one that was not changed in my previous PR but was adjusted in this one. Added both filebeat and auditbeat since they are compatible.